tests/plugins/apps/browser/test_chrome.py

```diff
@@ -348,3 +418,67 @@ def test_chrome_windows_snapshots(target_win_users: Target, fs_win: VirtualFiles
 
         # We map the same files in each of the snapshot directories.
         assert len(base_path_records) == len(snapshot_records)
+
+
+def test_chrome_windows_11_decryption(target_win_11_users_dpapi: Target, fs_win: VirtualFilesystem) -> None:
+    """Test if we can decrypt Windows 11 Google Chrome version 127/130 and newer passwords and cookies.
+
+    Elevation Service usage by Chromium-based browsers (Google Chrome, Microsoft Edge) depend on several environment
+    based circumstances (e.g. Windows version, feature flags, account log-in state). To force usage of the Elevation
+    Service, run the command(s) below.
+
+    .. code-block::
+
+        (chrome.exe|msedge.exe) --enable-features=UseElevator
+    """
+
+    keychain.KEYCHAIN.clear()
```

Comment:
Can you use the `guarded_keychain` fixture from `tests/helpers/test_keychain.py`? Can also move it to another `conftest.py` if you want.